src/utils/index.ts

```diff
@@ -136,3 +136,7 @@ export function divideCurrencyAmountByNumber(numerator: CurrencyAmount | undefin
 export function replaceUnderscoresWithSlashes(value: string) {
   return value.replace(/_/g, '/')
 }
+
+export function roundApr(apr: number) {
+  return apr > 1 ? Math.round(apr) : +apr?.toFixed(2)
```

Comment:
No need to coerce, since `apr: number`

```suggestion
export function roundApr(apr: number) {
  return apr > 1 ? Math.round(apr) : apr.toFixed(2)
```

Reply:
Need to convert since toFixed returns a string